### PR TITLE
Сhanged SNILS generator

### PR DESCRIPTION
**Pull request by Vlangf**
Сhanged SNILS generator, now it generates valid values ​​in accordance with the algorithm

### CODE REVIEW

File: CONTRIBUTORS.rst [hunk 1]
```diff
@@ -64,7 +64,8 @@ Contributors
 -  Simon `(DefaltSimon)`_
 -  dy `(duckyou)`_
 -  Arvind Prasanna `(aprasanna)`_
--  Pavel Kasyanov `(RaidoS)`_
+-  Pavel Kasyanov `(RaidoS)`_\
```
Comment (sobolevn) [inline CONTRIBUTORS.rst]: Extra `\`

File: tests/test_builtins/ru/test_russia_spec.py [hunk 2]
```diff
@@ -58,8 +58,20 @@ def test_generate_sentence(russia):
...
+    code = sum(numbers)
+    if code == 100 or code == 101:
+        assert result[9:] == '00'
+    if code < 100:
+        assert result[9:] == str(code)
+    if code > 101:
+        code = code % 101
+        if code == 100:
+            assert result[9 - 10] == '00'
+        print(result[9:] + '--' + '{:02}'.format(code))
```
Comment (sobolevn) [inline tests/test_builtins/ru/test_russia_spec.py]: Extra `print`

**Comment by Vlangf** [comment]
Thanks for you comments! I fixed some problems and made test more simply

**Comment by Vlangf** [comment]
Hello! If there are more comments, then I am ready to correct them. Sorry for the quality of the code, I did not start studying python very long ago

File: mimesis/builtins/ru.py [hunk 3]
```diff
@@ -88,18 +88,38 @@ def series_and_number(self) -> str:
...
+        for i in range(0, 9):
+            numbers.append(self.random.randint(0, 9))
+
+        for i in range(9, 0, -1):
+            control_codes.append(numbers[9 - i] * i)
+
+        control_code = sum(control_codes)
+        code = ''.join(str(number) for number in numbers)
+
+        if control_code == 100 or control_code == 101:
```
Comment (lk-geimfari) [inline mimesis/builtins/ru.py]: Why not `control_code in (100, 101)`?

**Pull request merged**